RoslynPluginGenerator/SonarQube.Plugins.Roslyn.PluginGenerator.csproj

```diff
@@ -65,7 +65,7 @@
       <Private>True</Private>
     </Reference>
     <Reference Include="NuGet">
-      <HintPath>..\packages\NuGet.CommandLine.3.3.0\tools\NuGet.exe</HintPath>
+      <HintPath>..\packages\NuGet.CommandLine.4.7.0\tools\NuGet.exe</HintPath>
```

Comment:
VS didn't fix up the _HintPath_ in the project file, so this is another change I had to make manually that I was expecting the UI to do for me.